autodocodec-api-usage/test/Autodocodec/Aeson/SchemaSpec.hs

```diff
@@ -78,6 +78,8 @@ spec = do
   jsonSchemaSpec @LegacyObject "legacy-object"
   jsonSchemaSpec @Ainur "ainur"
   jsonSchemaSpec @War "war"
+  jsonSchemaSpec @These "these"
+  -- jsonSchemaSpec @Expression "expression" -- Too slow
```

Comment:
It's not too slow, it has infinite runtime :p Fix the generator and this will work fine :D